pennylane/interfaces/batch/__init__.py

```diff
@@ -15,13 +15,137 @@
 This subpackage defines functions for interfacing devices' batch execution

```

Comment:
Is the contents of this file supposed to be accessible in the docs? I couldn't find it.

Reply:
I haven't made it accessible yet, was planning on doing a separate 'docs' PR once all the interfaces were in! This was partially to reduce the size of PRs